tests/ios/xcodeproj/post_build_check.sh

```diff
@@ -32,4 +43,52 @@ export EMPTY_SWIFTMODULE_FILES=`find build/Debug-iphoneos -name *".swiftmodule"
 for esf in ${EMPTY_SWIFTMODULE_FILES}; do
 	echo "Swiftmodule in build products dir at $esf is empty!";
 	exit 1;
-done
\ No newline at end of file
+done
+
+# LLDB tests
+# For each generated '.xcodeproj' check if the list of 'HEADER_SEARCH_PATHS' and
+# 'FRAMEWORK_SEARCH_PATHS' are present in the generated LLDB configuration file for each target
+#
+# If 'target.swift-extra-clang-flags' does not contain all those paths debugging '.swift' files
+# that depends on objc (via bridging headers for example) wont' work
+echo "Make sure all LLDB configuration files contain the expexted search paths"
+XCODE_PROJS=$(find . -name "$XCODE_PROJ.xcodeproj")
+for PROJ in $XCODE_PROJS; do
+    if [ $DESTINATION_TYPE = "simulator" ]; then
+        ALL_TARGETS=$(xcodebuild -project $PROJ -sdk iphonesimulator -alltargets -showBuildSettings | grep "TARGET_NAME" | sed -e "s/TARGET_NAME = //g" | xargs)
+    else
+        ALL_TARGETS=$(xcodebuild -project $PROJ -sdk iphoneos -alltargets -showBuildSettings | grep "TARGET_NAME" | sed -e "s/TARGET_NAME = //g" | xargs)
+    fi
+
+    for TARGET in $ALL_TARGETS; do
+        if [ $DESTINATION_TYPE = "simulator" ]; then
+            XCSETTINGS=$(xcodebuild -project $PROJ -sdk iphonesimulator -target $TARGET -showBuildSettings)
+        else
+            XCSETTINGS=$(xcodebuild -project $PROJ -sdk iphoneos -target $TARGET -showBuildSettings)
+        fi
+        HSP=$(echo "$XCSETTINGS" | grep "^[ ]*HEADER_SEARCH_PATHS = " | sed -e "s/HEADER_SEARCH_PATHS = //g" | xargs)
+        FSP=$(echo "$XCSETTINGS" | grep "^[ ]*FRAMEWORK_SEARCH_PATHS = " | sed -e "s/FRAMEWORK_SEARCH_PATHS = //g" | xargs)
+        PROJ_BASENAME=$(basename $PROJ .xcodeproj)
+        LLDB_CONFIG=$(find build -name "*$TARGET.lldbinit" | grep $PROJ_BASENAME.build)
```

Comment:
because of the specification for what we are looking for here, i suggest to add a README here to talk about example of what we are looking for inside .lldbinit file

Reply:
Will do!